roles/download/defaults/main.yml

```diff
@@ -386,9 +386,15 @@ calicoctl_binary_checksums:
     v3.15.2: 49165f9e4ad55402248b578310fcf68a57363f54e66be04ac24be9714899b4d5
 
 helm_archive_checksums:
-  arm: ca8792da269b72235987ea7245d1450a859b2c0658f591737d74b6c56cd9b1fa
-  amd64: 3fff0354d5fba4c73ebd5db59a59db72f8a5bbe1117a0b355b0c2983e98db95b
-  arm64: 87811b648ed9f4c84d3cb67bbea9b666bb7f6dd0ff6aca148b65f91058f73953
+  arm:
+    '3.5.0': ca8792da269b72235987ea7245d1450a859b2c0658f591737d74b6c56cd9b1fa
+    '3.5.1': 0b86a5a68df7376484babb6d7ffe1bae36012b4d65f1bcddb4255fb3bbe811db
```

Comment:
CI test jobs are failed like
```
{{ helm_archive_checksums[image_arch][helm_version] }}: 'dict object' has no attribute 'v3.5.1'
```
So we need to change here like
```
'v3.5.0': ca8792da269b72235987ea7245d1450a859b2c0658f591737d74b6c56cd9b1fa
'v3.5.1': 0b86a5a68df7376484babb6d7ffe1bae36012b4d65f1bcddb4255fb3bbe811db
```
by adding `v`

Reply:
@oomichi Should be ok now ...